tests/sbc_tests.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
```

Comment:
Looking at the `tests` folder, all tests are named `foo_test.py`

Reply:
maybe you want to rename this file.

Reply:
good point, will change.